test/ci/buildkite/run_pipeline.sh

```diff
@@ -27,6 +27,9 @@ elif [ "${BUILDKITE_PIPELINE_NAME}" == "gpu" ]; then
 elif [ "${BUILDKITE_PIPELINE_NAME}" == "models-gpu" ]; then
    export NGRAPH_TF_BACKEND=GPU
    PIPELINE_STEPS+=" ${SCRIPT_DIR}/models-cpu.yml "
+elif [ "${BUILDKITE_PIPELINE_NAME}" == "models-vpu" ]; then
+   export NGRAPH_TF_BACKEND=MOVIDIUS
```

Comment:
There is no plugin called `MOVIDIUS` afaik. Are you sure this is testing the right backend?

Reply:
I got it from here: https://docs.openvinotoolkit.org/2021.1/openvino_docs_install_guides_installing_openvino_linux.html#run-a-sample

Reply:
Shouldn't it be `MYRIAD` according to that link?

Reply:
Yes, fixed